### ISSUE TITLE
user session is not "active" if passwordless root access is disabled

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#7689]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

R4.1

### Brief summary

`loginctl` reports user session as "State: online" instead of "State: active". This breaks (at least) access rights for devices with `TAG+"uaccess"`.
Activating session manually works only from root, from user it gives:
```
$ loginctl activate
Failed to issue method call: Access denied
```

### Steps to reproduce

1. Take Debian minimal template (might apply to others too)
2. Install `libpam-systemd` (otherwise sessions are not registered at all`
3. Create AppVM based on it and start it
4. Call `loginctl session-state` as a user in that VM

### Expected behavior

`State: active`

### Actual behavior

`State: online`

Reported by @andrewdavidwong

**Comment by andrewdavidwong** [comment]
As discussed with @marmarek, this seems to affect only Debian-minimal, not Fedora-minimal.

**Comment by DemiMarie** [comment]
I think it is better to add `libpam-systemd`; it’s a tiny package and the UX benefit is worth it.

**Comment by andrewdavidwong** [comment]
> I think it is better to add `libpam-systemd`; it’s a tiny package and the UX benefit is worth it.

I disagree. The _raison d'être_ of minimal templates is to be _minimal_. "Tiny" and "UX benefit" are not principled reasons to add a package to a template the primary purpose of which is to be as minimal as possible. If we added every tiny package with a UX benefit, the template would no longer be minimal at all. It would be quite bloated, and there would be little reason for it to exist in addition to the standard template. Users are free add whichever packages they choose for such reasons, which is how minimal templates are intended to be used.

**Comment by unman** [comment]
Qubes 4.2 debian-12-minimal has package installed, but still shows this behavior

**Comment by ben-grande** [comment]
> fedora-37-minimal ships with systemd-libs already installed. qube based on that template shows "State: active" for `loginctl session-status`

Same on debian-13-minimal. `libpam-systemd` is being installed.
